docs/content/styles/components/Swatch.js

```diff
@@ -47,12 +47,12 @@ const ColorWrapper = styled('div')`
 `;
 
 const Swatch = ({ colorName }) => (
-  <ThemeProvider theme={theme.standard}>
+  <ThemeProvider theme={themes.standard}>
```

Comment:
Might as well switch to `themes.circuit`.

Reply:
We've been using `standard` in most places. Did we ever make a final decision between the two?